staging/cse/windows/configfunc.ps1

```diff
@@ -96,6 +96,7 @@ function Adjust-DynamicPortRange()
         Invoke-Executable -Executable "netsh.exe" -ArgList @("int", "ipv4", "add", "excludedportrange", "udp", "30000", "2768") -ExitCode $global:WINDOWS_CSE_ERROR_SET_UDP_EXCLUDE_PORT_RANGE
     } else {
         Invoke-Executable -Executable "netsh.exe" -ArgList @("int", "ipv4", "set", "dynamicportrange", "tcp", "33000", "32536") -ExitCode $global:WINDOWS_CSE_ERROR_SET_TCP_DYNAMIC_PORT_RANGE
+        Invoke-Executable -Executable "netsh.exe" -ArgList @("int", "ipv4", "set", "dynamicportrange", "udp", "33000", "32536") -ExitCode $global:WINDOWS_CSE_ERROR_SET_UDP_EXCLUDE_PORT_RANGE

```

Comment:
I think the "dynamicportrange" should be "excludeportrange"?

Reply:
we set `dynamicportrange` to exclude those ports under 33000.

Reply:
Update the error code